app/mm-redux/selectors/entities/roles.ts

```diff
@@ -154,7 +156,7 @@ export const getMyChannelPermissions = reselect.createSelector(
                     for (const permission of roles[roleName].permissions) {
                         permissions.add(permission);
                     }
-                    roleFound = true && teamRoleFound;
+                    roleFound = true && (teamRoleFound || !teamId);
```

Comment:
I'm not 100% clear why we do this. Why can't we remove the true?

Reply:
we can... not sure why i had it written like that